### ISSUE TITLE
Students not moving from one class to another class

### ISSUE THREAD

**Issue by dansputy** [JayVora-SerpentCS/OdooEduERP#58]
I have sequencially created classes and created different academic year. When I run the move standard wizard no student was moved despite the student has pass exam. 
We need to urgently pay attention to this or make alternative way to manually promote student from one standard to another.

Thank you for the great job.

**Comment by Arpit-SerpentCS** [comment]
@dansputy Plsease give me more details about the issue. How you have created the standards, class and exam of student so that I can check the issue accordingly in my system locally.

**Comment by dansputy** [comment]
I created two classes JSS1 with sequence 1, JSS2 with sequence 2, Created
subjects assigned them to classes, created a student, created exam
schedule, created exam timetable, enroll the student into JSS1, I also
created two Academic year, 2018 and 2019. The student took exam and pass
the subjects.

So to promote the student to JSS2, I clicked on move standard wizard, a pop
up came up asking to select academic year.

I did all that but to no luck.
